src/tests/multihost/admultidomain/conftest.py

```diff
@@ -0,0 +1,785 @@
+
```

Comment:
This was copied over from another directory/test right?  Was anything new added to it for the admultidomain tests?

Reply:
It was, I cleaned it up, adchildjoin was added.